rfc/ipni-w3c.md

```diff
@@ -2,53 +2,54 @@
 
 ## Authors
 
-- [gammazero], [olizilla]
+- [gammazero]
+- [gozala]
+- [hannah]
+
 
 ## Goals
 
 A user can use a multihash that identifies data stored on w3s to lookup the location where the data can be retrieved and content claims describing that data. w3s can update content claims if necessary without having to republish data multihashes.
 
-This RFC describes how IPNI indexing can be used to lookup data locations and content claims for any CID within a CAR stored in w3c.
+This RFC describes how IPNI indexing can be used to lookup data locations and content claims for any piece of data, identified by a multihash, stored in w3c.
 
 ## Abstract
 
-A user stores content as opaque blobs containing series of blocks addressed by multihashes. Currently, these blocks are collected into a CAR file for storage as one body of data.
-
-The user creates an external CAR index [CARx-index](https://github.com/web3-storage/RFC/pull/9/files) that contains tuples of (multihash, offset, length) of any items the user intends to make individually retrievable from the car.
+A user stores content as opaque blobs containing series of blocks addressed by multihashes. Currently, these blocks are collected into a CAR file for storage as one body of data. This will be generalized into a blob of data that does not require a CAR file.
 
-With an [inclusion claim](https://hackmd.io/@gozala/content-claims?utm_source=preview-mode&utm_medium=rec#Inclusion-Claims), the user who created the CAR asserts that CAR contains a given set of multihashes, transitively, via the CARx-index.
+The user creates an [inclusion claim] that contains the multihash of the data blob and an array of tuples of (multihash, offset, length) of any items the user intends to make individually retrievable from their data blob. This inclusion claim serves as an external index into the user's data that can be stored and retrieved separately from the data. This inclusion claim is also the user's assertion that their data blob contains the data represented by a given set of multihashes.
```

Comment:
> This inclusion claim serves as an external index into the user's data that can be stored and retrieved separately from the data

I've read this sentence several times and still confused by it. Specifically fact that it mensions **user's data** followed by **the data**. Could this be rephrased to make it bit more clear.

I think you're trying to say that indexed data can be retrieved independent from the blobs it's contained in ? Perhaps embracing term [IPLD Block](https://ipld.io/docs/intro/primer/#blocks-vs-nodes) could be of value to make things more concrete.

Reply:
Changing to...

> The user creates an [inclusion claim] that contains the multihash of the data blob and an array of tuples of (\
multihash, offset, length) of any blocks within the blob that the user intends to make individually retrievabl\
e. This inclusion claim serves as an external index into the user's data blob that can be stored and retrieved\
 separately from the user's data blob. The inclusion claim is also the user's assertion that their data blob c\
ontains the blocks represented by the set of multihashes in the inclusion claim.